Documentation/bpf.rst

```diff
@@ -833,7 +833,12 @@ failures:
 
 ::
 
-    Summary: 418 PASSED, 0 FAILED
+    Summary: 847 PASSED, 0 SKIPPED, 0 FAILED
+
+.. note:: Since Kernel Release 4.16 the bpf selftests has a dependency on LLVM >= 6.0
+          More info available on some mailing lists. (https://lwn.net/Articles/741773/)
+          Maybe you need to compile LLVM in the current version. More info at
+          http://llvm.org/docs/GettingStarted.html
```

Comment:
Thanks for the patch! Looks good. Could you also add a clarification why this is the case. The kselftests need this due to BPF to BPF calls. For users not making use of them, this dependency is not needed. Would be great if you could clarify that in the note a bit better so that people don't think LLVM>=6 is required for everything from now on. Thx